reposerver/repository/repository.go

```diff
@@ -1681,6 +1707,7 @@ func (s *Service) GetAppDetails(ctx context.Context, q *apiclient.RepoServerAppD
 
 		switch appSourceType {
 		case v1alpha1.ApplicationSourceTypeHelm:
+			log.Debug("Starting populateHelmAppDetails")
```

Comment:
Can this be dropped? Without additional context, I don't think it'll be helpful in a production environment.